js/plugins/mcp/README.md

```diff
@@ -13,53 +15,94 @@ To get started, you'll need Genkit and the MCP plugin:
 npm i genkit genkitx-mcp
 ```
 
-## MCP Client
+## MCP Client Manager
 
-To create an MCP client, you call the `mcpClient` function to generate a Genkit plugin for an MCP server. For example, to use MCP's example [filesystem server](https://github.com/modelcontextprotocol/servers/tree/main/src/filesystem):
+To connect to one or more MCP servers, you use the `createMcpClientManager` function. This function returns a Genkit plugin that manages connections to the configured MCP servers.
```

Comment:
Naming wise let's just go with `createMcpClient` and `createMcpManager` which is the multi-client manager.